collections/_testimonials/25-GOVIKING.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Numerical simulation of a cross-flow through a bundle to prepare the FSI simulations in the GO-VIKING European project"
```

Comment:
The title is quite long and is missing the actual role of preCICE, which was in flow coupling. How about:


```suggestion
title: "Turbulent flow coupling for flow-induced vibrations"
```